### PR TITLE
Add dialog-polyfill for browsers without dialog

### PR DESCRIPTION
**Pull request by bafolts**
This will add the dialog-polyfill for browsers that don't support dialog.

### CODE REVIEW

File: src/styles/common.less [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@import "./nes.min.less";
+@import (inline) "../../node_modules/nes.css/css/nes.min.css";
```
Comment (alrusdi) [inline src/styles/common.less]: If we are using the file from node_modules may be it is good idea to remove nes.min.less from our styles directory?

File: src/styles/common.less [hunk 2]
```diff
@@ -37,4 +36,4 @@ body {
 @import "./payments.less";
 @import "./venus.less";
 @import "./colonies.less";
-
+@import (inline) "../../node_modules/dialog-polyfill/dialog-polyfill.css";
```
Comment (alrusdi) [inline src/styles/common.less]: It is better to import it right after nes.min.css. Does it work there?

File: src/components/PlayerHome.ts [hunk 3]
```diff
@@ -15,6 +15,8 @@ import { Preferences } from "./Preferences"
 import { PlayerModel } from "../models/PlayerModel";
 import { Colony } from './Colony';
 
+const dialogPolyfill = require("dialog-polyfill");
```
Comment (alrusdi) [inline src/components/PlayerHome.ts]: Shouldn't it be initialized in top most component to be available project-wide? I mean that one from script.ts

Comment (bafolts) [reply src/components/PlayerHome.ts]: Webpack should only include the file once allowing us to import it this way. This avoids needing to pass the reference through `global` or finding it on the root node.

File: src/styles/common.less (hunk 1 above)
Comment (bafolts) [reply src/styles/common.less]: Yes, good call, I removed it.

**Pull request merged**